mon-pix/tests/unit/routes/login-pe_test.js

```diff
@@ -26,20 +26,20 @@ describe('Unit | Route | login-pe', function () {
       };
 
       // when
-      await route.afterModel(null, transition);
+      await route.beforeModel(transition);
 
       // then
       sinon.assert.calledWith(route.replaceWith, 'login');
```

Comment:
Suggestion : 
1. dans le `beforeEach`, donner une valeur de retour au stub de `replaceWith` :
```js
sinon.stub(route, 'replaceWith').returnsArg(0);
```
2. enlever le `await` et remplacer le `sinon.assert.calledWith` par une assertion plus intéressante sur la valeur de retour :
```suggestion
      const transition = route.beforeModel(transition);

      // then
      expect(transition).to.equal('login');
```

En faisant ainsi, non seulement on éviterait de traiter `replaceWith` comme un pur effet de bord (le test actuel passe même sans `return` dans `beforeModel`), mais de plus en vérifiant la valeur de retour de `beforeModel` *sans* utiliser `await`, on s'assurerait que la transition ne sera pas emballée dans une promesse (via `async` notamment) avec les conséquences qu'on connaît : https://github.com/emberjs/ember-test-helpers/issues/332#issuecomment-689596427